info.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "Bluegistics",
-  "version": "0.1.5",
+  "version": "0.1.6",
```

Comment:
don't worry about this, `script/update` will increment this and generate a new changelog entry automatically

Reply:
oh, haven't seen that one script! :D

Reply:
in the latest commit I reverted the version number to 0.1.5, but haven't run the update script. is that ok?